src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &:not(:nth-child(4n)) {
+    margin-right: $card-gap;
+  }
+
+  &:nth-child(-n+4) {
+    margin-bottom: 46px;
+  }
```

Comment:
It's better to write styles, connected to element positioning in your main.scss file, because it's not inner style of the card block and you shouldn't set margins for a block.